src/autoscaler/cf/app.go

```diff
@@ -83,10 +84,22 @@ func (c *cfClient) SetAppInstances(appId string, num int) error {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusCreated {
-		err = fmt.Errorf("failed setting application instances: %s [%d] %s", url, resp.StatusCode, resp.Status)
+		respBody, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			c.logger.Error("failed to read response body while setting app instance", err)
```

Comment:
add app id info to log.  
It is better the first argument does not include any space, so that index can be built on the whole string.